### ISSUE TITLE
p-based loss function

### ISSUE THREAD

**Issue by T-nfd** [zengh5/PRNU_CNN#2]
Dear @zengh5 

I read your great paper, "A comparison study of CNN denoisers on PRNU extraction."
I have a model that I've defined as `criterion = nn.CrossEntropyLoss().to(device)`.
So in the training part I have `loss = criterion(outputs, labels)`
I want to use your `p-based loss function,` but I don't know how?
Could you please help me with that?

**Comment by zengh5** [comment]
Dear T-nfd:
       Thanks for your interesting. Can you explain your issue more clearly and send it to my email zengh5@mail2.sysu.edu.cn

**Comment by JhxxxQ** [comment]
Hello @T-nfd , I doing some researches about sci too, could you leave your email for communication?
Thank you!
